src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -370,7 +371,7 @@ private static long ParseTimeValue(JToken jToken, string claimName)
                     return (long)resultDouble;
             }
 
-            throw LogHelper.LogExceptionMessage(new FormatException(LogHelper.FormatInvariant(LogMessages.IDX14300, claimName, jToken.ToString(), typeof(long))));
+            throw LogHelper.LogExceptionMessage(new FormatException(LogHelper.FormatInvariant(LogMessages.IDX14300, LogHelper.MarkAsNonPII(claimName), jToken.ToString(), LogHelper.MarkAsNonPII(typeof(long)))));
```

Comment:
>claimName [](http://example.com/codeflow?start=139&length=9)

what is claimName in this context?

Reply:
In this context claimName can be JwtRegisteredClaimNames.Iat or JwtRegisteredClaimNames.Nbf or JwtRegisteredClaimNames.Exp